README.md

```diff
@@ -1,6 +1,9 @@
 # http-request-signature
 An HTTP signature parser and generator fully compliant with the proposed "Signing HTTP Messages" [Internet Draft](https://www.ietf.org/id/draft-cavage-http-signatures-07.txt).
 
+## Status
+[![npm version][npm-image]][npm-url] [![build status][travis-image]][travis-url]
```

Comment:
Markdown makes it a little weird to nest imagages inside links, but at least the outermost link can use the usual `[text](url)` syntax; that may make this source line a little bit more readable:

```suggestion
[![npm version][npm-image]](npm-url) [![build status][travis-image]](travis-url)
```

(I tested using round parenthesis in the inner image link as well, but that doesn't seem to work, unfortunately.)

Reply:
👍

Reply:
I've tried that solution using the github editor + preview and that solution did not work. The variables `npm-url` and `travis-url` were not replaced by their values.

Reply:
Ah, indeed — I should have verified that. Sorry for the noise! 😅